examples/ChilloutMix-4.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Let's reorder the 3 images for ChilloutMix.
Use the duck image as the first one, and next the girl, and next the scenery. Just select 3